components/forms/Stopwatch.js

```diff
@@ -57,8 +60,13 @@ export default function Stopwatch({ setStoppedTime, stoppedTime }) {
               localStorage.removeItem("startTime");
             }}
           >
-            Stop
-          </button>
+            <Image
+              src="/images/svgs/stop.svg"
+              width={30}
+              height={30}
```

Comment:
Which unit is {30}? Is it "30px"?

Reply:
written like this, it always means px. ...I think. U can not write any units inside the brackets

Reply:
but I replaced it with the fontsize property now